app/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelper.scala

```diff
@@ -64,12 +64,12 @@ class ConsignmentItemAnswersHelper(
   def additionalReferences: Seq[SummaryListRow] =
     getAnswersAndBuildSectionRows(AdditionalReferencesSection(houseConsignmentIndex, itemIndex)) {
       additionalReferenceIndex =>
-        additionalReferenceRow(houseConsignmentIndex, itemIndex, additionalReferenceIndex)
+        additionalReferenceRow(additionalReferenceIndex)
     }
 
-  def additionalReferenceRow(hcIndex: Index, itemIndex: Index, additionalReferenceIndex: Index): Option[SummaryListRow] =
+  def additionalReferenceRow(additionalReferenceIndex: Index): Option[SummaryListRow] =
```

Comment:
Can this go in its own class? Ideally none of these methods will take any arguments. Makes it a lot easier to see at which level they should exist